src/CircuitBreaker/src/Hystrix/AbstractCommand.cs

```diff
@@ -97,7 +97,7 @@ protected virtual ExecutionResult CommandResult
 
     public ExecutionResult.EventCounts EventCounts => CommandResult.Eventcounts;
 
-    public List<HystrixEventType> ExecutionEvents => CommandResult.OrderedList;
+    public IEnumerable<HystrixEventType> ExecutionEvents => CommandResult.OrderedList;
```

Comment:
This looks quite odd, given the property name indicates an ordered list instead of a (potentially lazily executed) sequence. `IReadOnlyList<>` seems more appropriate. The advantage it that it allows callers to index into the collection, obtain its count etc, while still protecting against mutation. If mutation is something that should be possible, `IList<>` would be better.

Reply:
I think you're right, `IEnumerable` is likely wrong and is definitely confusing. `IReadOnlyList` should work and makes more sense